openedxstats/apps/sites/urls.py

```diff
@@ -3,8 +3,8 @@
 
 app_name = 'sites'
 urlpatterns = [
-    url(r'^sites/all/$', views.ListView.as_view(), name='sites_list'),
-    url(r'^sites/add_site/$', views.add_site, name='add_site'),
+    url(r'^sites/all/$', views.ListView.as_view(), name='sites_list'), # Modified view to function based for experimenting, will convert to 
+    url(r'^sites/all/json$', views.SiteView_JSON, name='sites_list_json'), # Serves JSON data for all websites in database
```

Comment:
Commenting here is awkward.  If you want to explain the view function, put the description on it instead of on the URL.